### PR TITLE
Fix dropItemAtPosition

### PR DESCRIPTION
**Pull request by bruh-9000**

### CODE REVIEW

File: src/gameClasses/Unit.js [hunk 1]
```diff
@@ -1368,6 +1368,11 @@ var Unit = TaroEntityPhysics.extend({
 				item.setOwnerUnit(undefined);
 				item.setState('dropped', defaultData);
 
+				// Move item if dropping item at a position
+				if (position) {
+					item.teleportTo(position.x, position.y, item._rotate.z);
```
Comment (sloont) [inline src/gameClasses/Unit.js:1373]: is there a difference between this rotate value and `item.anchoredOffset.rotate` used above?

File: src/gameClasses/components/script/ActionComponent.js [hunk 2]
```diff
@@ -159,8 +159,8 @@ var ActionComponent = TaroEntity.extend({
 								}
 							}
 							break;
-						}
-
+						}
```
Comment (sloont) [inline src/gameClasses/components/script/ActionComponent.js:162]: remove this whitespace and the whitespace on the line below

**Review by sloont — CHANGES_REQUESTED** [review]

File: src/gameClasses/Unit.js (hunk 1 above)
Comment (bruh-9000) [reply src/gameClasses/Unit.js:1373]: Sorry I don't understand what you mean.

Comment (bruh-9000) [reply src/gameClasses/Unit.js:1373]: I'm not sure

**Comment by bruh-9000** [comment]
@sloont

**Review by Grenagar — APPROVED** [review]

**Pull request merged**